trino/trino_test.go

```diff
@@ -321,6 +321,23 @@ func TestExec(t *testing.T) {
 	}
 }
 
+func TestUnsupportedHeader(t *testing.T) {
```

Comment:
Can you also add and integration test calling out to 
* `SET SESSION grouped_execution=true`
* `SET ROLE dummy`
* `SET PATH dummy`
* `RESET SESSION grouped_execution` (not sure if this one will work; may not send X-Presto-Clear-Session header if it was not set before, which we cannot do)

Reply:
Done! Actually only `SET PATH` failed since there's some client capability negotiation mechanism, this client doesn't report any capabilities and setting PATH is not allowed by default.